utils/ipmitool/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ipmitool
+PKG_VERSION:=1.8.17
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=@SF/ipmitool
+PKG_MD5SUM:=f7408aa2b40333db0413d4aab6bbe978
+PKG_MAINTAINER:=Anton D. Kachalov <mouse@yandex-team.ru>
+
+PKG_LICENSE:=SUN
```

Comment:
Not sure that's the right license tag.  These are meant to be to  the SPDX tags, see: https://spdx.org/licenses/